_includes/templates/forms/plain-writing-feedback-2-form.html

```diff
@@ -0,0 +1,294 @@
+                            <option value="WA">Washington</option>
+                            <option value="WV">West Virginia</option>
+                            <option value="WI">Wisconsin</option>
+                            <option value="WY">Wyoming</option>
+                        </select>
+                    </div>
+                </div>
+            </div>
+
+            <div class="form-group">
+                <div class="form-l_col-1-2">
+                    <label class="form-label-header" for="plain-writing-feedback_zip-code">
+                        Zip code
+                    </label>
+                </div>
+                <div class="form-l_col-1-2">
+                    <input class="input__long"
+                           id="plain-writing-feedback_zip-code"
+                           type="text"
+                           placeholder="Requestor's zip code"
+                           required>
+                </div>
+            </div>
+
+           <div class="form-group">
+                <div class="form-l_col-1-2">
+                    <label class="form-label-header" for="plain-writing-feedback_comment-date">
+                        Comment date
+                    </label>
+                </div>
+                <div class="form-l_col-1-2">
+                    <input class="input__long"
+                           id="plain-writing-feedback_comment-date"
+                           type="number"
+                           placeholder="mm/dd/yyyy"
+                           required>
+                </div>
+            </div>
+
+            <div class="form-group">
+                <div class="form-l_col-1-2">
+                    <label class="form-label-header" for="plain-writing-feedback_comments">
+                        Plain writing feedback or comments
+                    </label>
+                </div>
+                <div class="form-l_col-1-2">
+                    <textarea class="input__long"
+                              id="plain-writing-feedback_comments"
+                              required>
+                    </textarea>
+
+                </div>
+            </div>
+
+            <div class="form-group">
+                <div class="form-l_col-1-2">
+                    <label class="form-label-header" for="plain-writing-feedback_url">
+                        URL of page with plain writing issue
+                    </label>
+                </div>
+                <div class="form-l_col-1-2">
+                    <input class="input__long"
+                           id="plain-writing-feedback_url"
+                           type="text"
+                           placeholder="(Optional)"
+                           >
+                </div>
+            </div>
+
+            <div class="form-group">
+                <div class="form-l_col-1-2">
+                    {# Empty column to align buttons with input fields #}
```

Comment:
This shouldn't be necessary. Every grid system out there has a push-pull syntax, we should look into adding one.